Resources/ServerInfo/Guidebook/Engineering/TEG.xml

```diff
@@ -21,19 +21,153 @@
+  - The Pipe segment is where the burn actually occurs; to start it off, one can use a heater to increase the temperature up to the ignition temperature of Plasma. Afterwards, the reaction should be self-sustaining, so long as the Pressure and Moles supplied remains high enough. Be warned; if you wish to remove the heater, it will carry some of this superheated gas with it, transferring it to the next pipenet you connect it to. Best to space the gas through a space vent, if you must.
+  - The Low-pressure pump (whose pressure should be [italics]slightly lower[/italics] than the input pump) prevents [italics]all[/italics] the gas from passing through the circulator, which could result in the loss of the Moles required to sustain a burn.
+  - The Circulator is where this generated heat will flow to the cold loop; afterwards, feel free to space the waste gases.
+
+  Note: Pressure pumps are used here as, while they pump on pressure (not flow-rate, which is comparatively faster), they are a bit easier to control when it comes to the limited Plasma supply on-station. However, the steps shown can be followed with volumetric pumps too.
+
+  ## The Burn Chamber
+
+  The burn chamber is the preferred method for heating up gases, and it is commonly used for other purposes too. (see: Tritium production)
+
+  Most (if not all) stations have the burn chamber separated from the main atmospherics block by a 1-wide spaced grid, presumably to prevent conduction. The chambers consist of 3(+1) important parts:
+  - The Air Injector/Passive Vent
+  - The Space Vent
+  - The Air Scrubber (array)
+  - The Air Alarm (Optional, but Recommended)
+
+  The air injector (or Passive Vent) injects air (or allows air to flow) into the burn chamber. Either should be supplemented with a pump before it, to keep pressures high.
+  There are a few notable differences between the passive vent and the air injector; mostly speaking, the air injector allows you to keep pumping in air at higher pressures, at the expense of being mildly more tedious to work with than a passive vent. If you are using the space vent, it really doesn't matter which one you use.
+
+  The space vent (designated as a blast door to space on one side of the burn chamber) allows waste gases to be expelled and destroyed. This is only really useful if you want to produce tritium and you want to keep the burn going, but you can open this if you want to for now.
+
+  The air scrubber(s) serve to input the now hot air into a pipe that should lead to the hot-side Circulator. They also 'filter' the air; this is only really useful if you want to filter out a useful gas (usually, tritium). However, the default settings also are not the best, as gas intake is low, and unburnt plasma is also filtered out, both reducing the efficicency of the burn. See the Air Alarm step for more information.
```

Comment:
radiators to plasma/frezon loop are better for heat transfer than scrubbers (which will have to be manually configured to scrub oxygen too) and trying to use teg for trit production is useless, make a suitable setup instead of getting at most a few mols while taking hours off the fuel supply (if you even fill the chamber with scrubbers)

Reply:
will do, cutting out the trit-production part